src/routes/user.js

```diff
@@ -4,20 +4,28 @@ import userController from '../controllers/userController';
 
 const {
   userValidator: {
-    signUpValidator, loginValidator, profileValidator
+    signUpValidator, loginValidator, profileValidator, editProfileValidator
   },
   verifyToken
 } = middlewares;
-const { signUp, login, getProfile } = userController;
+const {
+  signUp, login, getProfile, editProfile
+} = userController;
 const user = express.Router();
 
+const USER_URL = '/users';
+const PROFILE_URL = '/profiles';
+
 // Route to sign up a user
-user.post('/users', signUpValidator, signUp);
+user.post(`${USER_URL}`, signUpValidator, signUp);
 
 // Route to login a user
-user.post('/users/login', loginValidator, login);
+user.post(`${USER_URL}/login`, loginValidator, login);
 
 // Route to get user profile by userId
-user.get('/profiles/:userId', verifyToken, profileValidator, getProfile);
+user.get(`${PROFILE_URL}/:userId`, verifyToken, profileValidator, getProfile);
+
+// Route to edit a user profile
+user.patch(`${PROFILE_URL}`, verifyToken, editProfileValidator, editProfile);
```

Comment:
which user is allowed to access this endpoint?

Reply:
Only authorized users can access this route, to update their own profile only